models/adapters/adwords_ad_performance_adapter.sql

```diff
@@ -1,8 +1,11 @@
 {% if var('adapter_value') == 'url' %} 
```

Comment:
this model seems to still only be stitch-specific, right? from what I can tell, there is only one version of this model (the stitch one).

This is actually a "transform" model--I don't think it touches the raw data. So I think the right answer is to implement it in your "router" pattern. I'm not 100% sure here, but happy to discuss with you.

Reply:
@jthandy Check out the latest commit. I moved the logic to the routers, _however_ `fivetran/adwords_criteria_performance` is now identical to `fivetran/adapters/criteria/adwords_performance_adapter`. Let me know what you think the right step forward is.